tests/data/shape.catalog.yaml

```diff
@@ -9,10 +9,39 @@ sources:
       urlpath: '{CATALOG_PATH}/stations/stations.shp'
 
   MEOW:
-      description: MEOW
-      driver: intake_geopandas.geopandas.ShapefileSource
-      args:
-        urlpath: http://maps.tnc.org/files/shp/MEOW-TNC.zip
+    description: MEOW
+    driver: intake_geopandas.geopandas.ShapefileSource
+    args:
+      urlpath: http://maps.tnc.org/files/shp/MEOW-TNC.zip
```

Comment:
If they aren't too large (< ~1MB) and there are no licensing difficulties, it may be worthwhile to copy these shapefiles into this repository so that we have better control over the stability of URLs.

Reply:
I am unsure about the licence there. worked with this a year ago. seems pretty stable.